src/main/scala/com/fulcrumgenomics/util/NcbiRefSeqGffSource.scala

```diff
@@ -259,7 +262,15 @@ class NcbiRefSeqGffSource private(lines: Iterator[String],
       case xs    => (Some(xs.minBy(_.start).start), Some(xs.maxBy(_.end).end))
     }
 
-    Transcript(txRec("transcript_id"), chrom, txRec.start, txRec.end, cdsStart, cdsEnd, txRec.negative, exons)
+    // Figure out the ID of the transcript-like row
+    val id = if (txRec.has("transcript_id")) txRec("transcript_id") else txRec("ID").split('-').drop(1).mkString("-")
```

Comment:
This is done because the `ID` field is forced to be file-wide unique, and so names get prefixed with silly things. E.g. for the tRNA I've added to the test GFF, the ID field is `ID=rna-TRN-GTT5-1`.  Other types get `id-` prepended or other not-useful strings.

Reply:
I am a bit confused.  Where does _fgbio_ prepend a field to the transcript ID besides the tests?  Or are you now requiring the user to have a some field before the "-" to make it work when `transcript_id` is not not found?

Reply:
fgbio doesn't prepend anything ... RefSeq does.  Perhaps a couple of examples will be helpful?  Apologies that this is a bit hard to read in github:

```
NC_000001.11  Gnomon       gene        89006     267354    .  -  .  ID=gene-LOC100996442;Dbxref=GeneID:100996442;Name=LOC100996442;gbkey=Gene;gene=LOC100996442;gene_biotype=misc_RNA
NC_000001.11  Gnomon       transcript  89006     267354    .  -  .  ID=rna-XR_001737582.2;Parent=gene-LOC100996442;Dbxref=GeneID:100996442,Genbank:XR_001737582.2;Name=XR_001737582.2;gene=LOC100996442;transcript_id=XR_001737582.2
...
NC_000001.11  tRNAscan-SE  gene        16520585  16520658  .  -  .  ID=gene-TRN-GTT5-1;Dbxref=GeneID:107985757,HGNC:HGNC:38582;Name=TRN-GTT5-1;gbkey=Gene;gene=TRN-GTT5-1;gene_biotype=tRNA
NC_000001.11  tRNAscan-SE  tRNA        16520585  16520658  .  -  .  ID=rna-TRN-GTT5-1;Parent=gene-TRN-GTT5-1;Dbxref=GeneID:107985757,HGNC:HGNC:38582;gene=TRN-GTT5-1;inference=COORDINATES: profile:tRNAscan-SE:1.23;product=tRNA-Asn
```

This is two genes and their "transcript" records that come next in the file.  You can see that the first "transcript" has `transcript_id=XR_001737582.2` and also `ID=rna-XR_001737582.2`.  That's just how it is in the file - I think they do this so that e.g. if a gene and a transcript have the same ID, they still end up file-wide unique because each level/type gets a different prefix like `rna-`.
The second "transcript" doesn't have `transcript_id` defined, so I need to do something else.  In this case stripping off the `rna-` and yielding `TRN-GTT5-1` from `ID=rna-TRN-GTT5-1` is about the most sensible thing I could see to do. 

If that doesn't make sense perhaps we should discuss live.

Reply:
That makes sense, can you add a note in the docs that the id is transformed?